src/aat/java/uk/gov/hmcts/ccd/datastore/tests/helper/idam/OAuth2.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.ccd.datastore.tests.helper.idam;
+
+import uk.gov.hmcts.ccd.datastore.tests.Env;
+
+public enum OAuth2 {
+
+    INSTANCE;
+
+    private final String clientId;
+    private final String clientSecret;
+    private final String redirectUri;
+
+    OAuth2() {
+        clientId = Env.require("OAUTH2_CLIENT_ID");
```

Comment:
same as above we could use Validate.notNull

Reply:
Now using Validate.notNull